src/components/Home/Testimonials.jsx

```
@@ -82,99 +83,84 @@ const Testimonials = () => {
-                      <div
-                        className="text-sm sm:text-base md:text-lg lg:text-[19px]"
-                        dangerouslySetInnerHTML={{ __html: testimonialsData[cardIdx].paragraph }}
-                      />
-                    </div>
-                  </motion.div>
-                </AnimatePresence>
-              );
-            })}
-          </div>
+                      className="text-[19px] sm:text-[18px] md:text-[19px]"
+                      dangerouslySetInnerHTML={{ __html: testimonialsData[cardIdx].paragraph }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)